### DISCUSSION TITLE
Dark frame toggle for FITS

### DISCUSSION THREAD

**Discussion by gigaklatch** [aaronwmorris/indi-allsky#1205] [Ideas]
Would it be possible to add the option of toggling dark frames on or off in "Process FITS"? This would really help in making before and after comparisons, seeing if the dark frames actually do improve anything. Perhaps this function is already there?

**Comment by gigaklatch** [comment]
Alright, I will assume that this is not possible as it was mentioned in a different thread that the dark frames are applied to the raw data **before** it is saved as a fits file.

**Reply by aaronwmorris** [reply]
Under the current model, it's not possible due to what you already mentioned.  The FITS data already has the dark frame subtracted.

I could add an option to temporarily disable dark frame calibration, if you wanted to just have the option.  But I am still think of how I could add the calibration option.

**Comment by aaronwmorris** [comment]
Once again, I might have been overthinking things.

#1240 includes options to disable dark frame calibration entirely, or just disable dark frame calibration for FITS files.  There is also now an option in the FITS processor to apply dark calibration frames to test removing the hot pixels.

**Reply by gigaklatch** [reply]
Now I have the latest INDI-allsky installed(actually started from scratch on a reformatted SSD-drive), almost everything works well so far, except for when I want to process the FITS files  and try out differens settings. It just says "Processing..." and seemingly never finishes doing so, It shows neither the unprocessed fits or the processed ones with the settings applied. I can download the FITS files and these can be viewed without any problems.

What might be wrong?

**Reply by aaronwmorris** [reply]
Looks like I introduced a bug recently.  Fixed in #1285 .  Thank you for reporting this.
